File: GO/class2/ex3_rectangle.go
/*Escreva um programa em C que calcule o perímetro e a área de um
retângulo de base 10 e altura 5 e imprima os resultados na tela.*/

package main

import "fmt"

func main() {

	base := 10
	height := 5

	perimeter := 2*base + 2*height

	area := base * height

	fmt.Printf("Perimetro: %d\n", perimeter)
	fmt.Printf("Area: %d\n", area)
}

File: C/class9/ex46_redoTheQuestions/class2/ex3_rectangle.c
/*Escreva um programa em C que calcule o perímetro e a área de um
retângulo de base 10 e altura 5 e imprima os resultados na tela.*/

#include <stdio.h>

int main() {

    int base = 10;
    int height = 5;
    int perimeter;
    int area;

    perimeter = 2 * base + 2 * height;

    area = base * height;

    printf("Perimetro: %d\n", perimeter);
    printf("Area: %d\n", area);

    return 0;
}

